app/src/main/java/com/example/keybindhelper/ui/Projects/ProjectsFragment.kt

```diff
@@ -82,27 +82,27 @@ class ProjectsFragment : Fragment() {
         }
         )
 
-        return root;
+        return root
 
     }
     fun openKeybindFragment(){
         NavHostFragment.findNavController(this).navigate(R.id.nav_keybind)
     }
-    fun RefreshProjectList():List<Project>{
-        val rv= root.findViewById<RecyclerView>(R.id.projects_recycler);
+    fun refreshProjectList():List<Project>{
+        val rv= root.findViewById<RecyclerView>(R.id.projects_recycler)
         val projects=DatabaseManager.getOrderedProjects()
-        val searchBox=root.findViewById<EditText>(R.id.projects_search_edit);
-        if(searchBox.text.length>0) {
-            val filteredP= mutableListOf<Project>();
-            val search=searchBox.text.toString().lowercase();
+        val searchBox=root.findViewById<EditText>(R.id.projects_search_edit)
+        if(searchBox.text.isNotEmpty()) {
+            val filteredP= mutableListOf<Project>()
+            val search=searchBox.text.toString().lowercase()
             for (project in projects) {
                 if(project.name.value!!.lowercase().contains(search))
-                filteredP+=(project);
+                filteredP.plusAssign(project)
```

Comment:
Augmented assignments. Converted "+=" to ".plusAssign".
https://kotlinlang.org/docs/operator-overloading.html#augmented-assignments

Reply:
Huh, neat.